### DISCUSSION TITLE
Uncaught ReferenceError: Buffer is not defined

### DISCUSSION THREAD

**Discussion by ThisIsRex** [exceljs/exceljs#1726] [Q&A]
I tried to bundle ExcelJS using rollup and got an error: `Uncaught ReferenceError: Buffer is not defined`
It seems ExcelJS uses a Buffer class which is not present in the browser
How to resolve this error?
rollup config:
```js
  import { nodeResolve } from '@rollup/plugin-node-resolve';
  import commonjs from '@rollup/plugin-commonjs';
  import builtins from 'rollup-plugin-node-builtins';
  
  export default {
      input: "./src/main.js",
      output: [{ file: "./dist/core.js", format: "iife", intro: 'const global = window;' }],
      plugins: [commonjs(), builtins(), nodeResolve()],
  
  };
```

**Comment by jperez-neostella** [comment]
Hi, I'm having the same issue. I use MUI X which uses exceljs, and Vite works in development, but can´t build the production bundle.
`error during build:
SyntaxError: Unexpected token (11:63288) in /home/PATH/node_modules/exceljs/dist/exceljs.min.js
    at Parser.pp$4.raise (file:///home/PATH/node_modules/rollup/dist/es/shared/rollup.js:19614:13)
`
Have you found any solution?
